app/controllers/application_controller.rb

```diff
@@ -3,6 +3,10 @@ class ApplicationController < ActionController::Base
   before_action :configure_permitted_parameters, if: :devise_controller?
   rescue_from ActiveRecord::RecordNotFound, with: :record_not_found
   
+  
```

Comment:
please remove these extra line, there should not be more than one consecutive blank line in code.